Documentation/configure-kubectl.md

```diff
@@ -15,15 +15,15 @@ After downloading the binary, ensure it is executable and move it into your `PAT
 
 ```sh
 $ chmod +x kubectl
-$ mv kubectl /usr/local/bin/kubectl
+$ sudo mv kubectl /usr/local/bin/kubectl
```

Comment:
Can we drop the sudo? When it is specifically on a CoreOS machine, it's safer to make assumptions about underlying system (and when sudo is necessary). But in this case it's their local machine, and I'd rather leave this up to user.